### ISSUE TITLE
[Sky Island]  Basement Locator Craft strangely difficult.

### ISSUE THREAD

**Issue by Aimless-dude** [CleverRaven/Cataclysm-DDA#77601]
### Is your feature request related to a problem? Please describe.

The basement locator upgrade, judging from when it unlocks and explanations of the gameflow in previous documentation, should be made from relatively easy to get and early game items. However, both the animal suits and the sets of tabletop games required for it are relatively rarer, especially when you consider all the variations of board games and puzzle sets can spawn instead of the exact ones you require.

### Solution you would like.

Either the requirements should be wider for these crafts should be wider, allowing more variations of table top games, or it should feature different, but still thematic items.

**Comment by Karol1223** [comment]
I've never had trouble with animal suits, but I do agree about the tabletop games. I kept finding different ones and had to double check the specific ones that I needed instead. That part felt awkward.

**Comment by Cupidatis** [comment]
Same for second mission per raid. Can't find platinum necklace a lot of time. Unlucky.

**Comment by Standing-Storm** [comment]
What's the appropriate solution, do you think? "Any 3 tabletop games" (make it a requirement list and put them all in there) and "any platinum jewelry"?

**Comment by Cupidatis** [comment]
@Standing-Storm for games maybe good decision. But jewelry... Someone couldn't find something else. Maybe more variables for all ingredients?

**Comment by Aimless-dude** [comment]
> What's the appropriate solution, do you think? "Any 3 tabletop games" (make it a requirement list and put them all in there) and "any platinum jewelry"?

Yes, I think that would work very well!. Oh, in a similar thing, one of the bunker upgrades asks for shovels, but doesn't accept entrenching tools or wood shovels.

**Comment by AlexMooney** [comment]
> What's the appropriate solution, do you think? "Any 3 tabletop games" (make it a requirement list and put them all in there) and "any platinum jewelry"?

I tried to implement this, but this sort of recipe is invalid:
```json
      [ [ "boardgame", 1, "LIST" ] ],
      [ [ "boardgame", 1, "LIST" ] ],
      [ [ "boardgame", 1, "LIST" ] ]
```
(as is the `"using"` equivalent), whereas this legal recipe requires 3 of the same game:
```json
      [ [ "boardgame", 3, "LIST" ] ]
```
Should we change it to be a set of 3 of a kind, or split the list into categories like `[ [ "boardgame_ancient", 1 ], [ "boardgame_classic", 1 ], [ "boardgame_modern", 1 ]` with ancient = cards, checkers, chess, mancala, go, etc; classic = pictionary, monopoly, etc; modern = catan, spirit_island, etc?
